src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -9,7 +9,6 @@ import DiscoveryTitleSeparator from '../elements/DiscoveryTitleSeparator';
 import DomainList from '../elements/DiscoveryItemList';
 import LoadingWithoutModal from '../../../components/LoadingWithoutModal';
 import RecentSearch from '../elements/RecentSearch';
-import useIsReady from '../../../hooks/useIsReady';
 import {COLORS} from '../../../utils/theme';
 import {Context} from '../../../context/Store';
 import {fonts} from '../../../utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useIsReady` hook import has been removed in the new hunk. Ensure that this hook is not used anywhere else in the file, otherwise it will cause a reference error.
```diff
- import useIsReady from '../../../hooks/useIsReady';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->